tardis/montecarlo/montecarlo_numba/numba_interface.py

```diff
@@ -43,6 +64,65 @@ def __init__(self, r_inner, r_outer, v_inner, v_outer, time_explosion):
         self.time_explosion = time_explosion
 
 
+@jitclass(geometry_grid_spherical_1d_spec)
+class GeometryGridSpherical1D(object):
+    def __init__(self, r_inner, r_outer):
+        """
+        1D spherical shell model
+
+        Parameters
+        ----------
+        r_inner : numpy.ndarray
```

Comment:
Can you add short descriptions of these variables?

Reply:
Done